### ISSUE TITLE
cpu-type set improperly

### ISSUE THREAD

**Issue by giacomoleopardo** [Piker-Alpha/ssdtPRGen.sh#289]
Hi Pike, as noticed before so many times, I encountered the Warning about Cpu-type while generating ssdt. 
![schermata 2016-10-29 alle 11 48 56](https://cloud.githubusercontent.com/assets/6927676/19828910/8a998ca0-9dd3-11e6-95f2-2e31f52774d2.png)

I injected in Clover config.plist/cpu/type the value 0x0905 as suggested by your script.
![schermata 2016-10-29 alle 11 49 47](https://cloud.githubusercontent.com/assets/6927676/19828911/8df95718-9dd3-11e6-9572-6c48f68a3ad1.png)
After reboot, sierra detect Cpu Intel i3
![schermata 2016-10-29 alle 11 51 40](https://cloud.githubusercontent.com/assets/6927676/19828914/a7f99dee-9dd3-11e6-9c1d-67748ff85347.png)

Specs:
- ASRock Fatal1ty Z170 Gaming-ITX/ac
- Intel® Core™ i7-6700 Processor 
- macOS Sierra 10.12.1 fresh install (no update)
- Clover Bootloader UEFI mode

**Comment by giacomoleopardo** [comment]
I actually don't know if this is a matter of ssdtPRGen.sh or Clover bootloader.

**Comment by Piker-Alpha** [comment]
No. This issue is not related to Clover, but errors in the script. It's on my TODO list. Like so many other things.

Thanks for the reminder!

**Comment by ghost** [comment]
Same issue with the Core M3 6Y30. CPU type provided by your script also makes it appear like Core i3. Thanks for your work!

**Comment by Piker-Alpha** [comment]
@OttoEdler,

The script injects nothing cpu-type related. All it does right now is to show a value that might be used as cpu-type, and some of them are simply wrong.

**Comment by Crazor** [comment]
For my i5 6600, the script wrongly suggests that the type should be 0x0605 instead of 0x0905, which makes it show up as an i3.

**Comment by locmer** [comment]
Same here GA-Z170N-WIFI + Core I7 6700k to become unstable when following the recommended:
Warning: 'cpu-type' may be set improperly (0x0705 instead of 0x0905)

**Comment by JrCs** [comment]
Definitively a "bug" in the script: report 0x0705 instead of 0x0905 for my Core i7 6700k

**Comment by VGerris** [comment]
hi, I have a Dell Precision 5520 with Xeon E3-1505M V6. I keep seeing unknown in about this Mac.
the script showed the warning for 0x might needing to be 0x09. I tried to override in the field shown in the screenshot and also use FakeCPuid without success. What is the proper way to fix it? by the way, the link pointing to the solution to check is dead.
